### ISSUE TITLE
Unsafe initialization of `words` array

### ISSUE THREAD

**Issue by rybnik** [Tessmore/sbd#11]
Hi folks, thank you for the handy script! I wanted to share a quick consideration:

* Under certain circumstances the value of the `words` [artifact](https://github.com/Tessmore/sbd/blob/master/lib/tokenizer.js#L57) will be `null` (e.g. whenever match is unable to find a proper match).
* [Because we are not checking that the value is properly defined](https://github.com/Tessmore/sbd/blob/master/lib/tokenizer.js#L64), under specific circumstances we will be invoking `length` on an `null` artifact yielding a `TypeError: Cannot read property 'length' of null`.

**Comment by Tessmore** [comment]
Hey,

Hmm interesting. `text` should at least be defined and have a length.  I agree it's useful to check for typeof string before starting to split it into words and sentences. But if you pass an empty array I get "TypeError: text.match is not a function". What circumstance(s) do you see to get `text.match() => null`?

**Comment by rybnik** [comment]
@Tessmore  c'mon!!! That was blazing fast! :) Thank you very much for looking into this. Since I have been poking around with your SBD I'll try to find some time to give back a little bit.

Thank you again!
